frontend/src/Screen2.js

```diff
@@ -128,7 +112,12 @@ const options = {
 }
 
 export const Screen2 = () => (
-  <CenterContent>
+  <Container
```

Comment:
Hmm, for these, it might be worth adding a preset to the Container, so we can just export and reuse CenterContainer. Doesn't have to be done for this PR, but it's an idea for a future PR :)